internal/processing/status/v1beta1/v1beta1.go

```diff
@@ -0,0 +1,136 @@
+package v1beta1
```

Comment:
Is it a benefit to have this in a separate v1beta1 package? Maybe it's better to move this and v2alpha1 status to the `status` package, especially since there is some duplicated code in these files that can be reused.